OpenAPI-v2.yml

```
@@ -662,7 +662,9 @@ paths:
                     default: false
                     description: If any error occurred during the request
                   response:
-                    $ref: '#/components/schemas/Event'
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/Event'
```

```suggestion
                      $ref: '#/components/schemas/EventIndex'
```

The original PR used the wrong schema reference, this endpoint does not provide a list of attending users/VTCs.